Sources/ApolloCodegenLib/Frontend/CompilationResult.swift

```diff
@@ -31,6 +31,21 @@ public class CompilationResult: JavaScriptObject {
     override public var debugDescription: String {
       "\(name) on \(rootType.debugDescription)"
     }
+
+    lazy var nameWithSuffix: String = {
```

Comment:
Moved out of `OperationDefinitionTemplate` so the file generator can use the logic too. Also lazy because it'll be called more than once.